lib/templates/facebook/koala.rb

```diff
@@ -0,0 +1,9 @@
+Koala.configure do |config|
+  config.access_token = ENV['FACEBOOK_SECRET']
+  config.app_id = ENV['APP_ID']
```

Comment:
I think we should name this `FACEBOOK_APP_ID`. The API can potentially use several app IDs and it may get confusing.